plugins/kinet/KiNetNode.cpp

```diff
@@ -18,23 +18,32 @@
  * Copyright (C) 2013 Simon Newton
  */
 
+#include <algorithm>
 #include <memory>
 
 #include "ola/Constants.h"
 #include "ola/Logging.h"
 #include "ola/network/IPV4Address.h"
 #include "ola/network/SocketAddress.h"
+#include "ola/util/SequenceNumber.h"
 #include "plugins/kinet/KiNetNode.h"
 
 namespace ola {
 namespace plugin {
 namespace kinet {
 
+using ola::network::HostToNetwork;
 using ola::network::IPV4Address;
 using ola::network::IPV4SocketAddress;
 using ola::network::UDPSocket;
 using std::auto_ptr;
 
+/*
+ * Initialize packet counter.
+ */
+ola::SequenceNumber<uint32_t> KiNetNode::m_transaction_number = (
+  ola::SequenceNumber<uint32_t>());
+
```

Comment:
You can drop this and just do it in the header:
```suggestion
```

Reply:
It's a static variable, so I was getting errors if I did not define it outside the header.

Reply:
Fixed!